ExpressWash/ExpressWash/Car+Convenience.swift

```diff
@@ -16,47 +16,55 @@ enum CarSize: String {
 }
 
 extension Car {
-    convenience init(carId: Int32 = 0,
+    convenience init(clientId: Int32,
```

Comment:
I don't know why I didn't think of this when we were talking about it. The reason `clientId` wasn't included originally is that a Car has a relationship to a User called `owner`. We could change the `owner` relationship to `client` to match the backend, and then you get the ID by saying `car.client.userId`. You set this relationship when instantiating a new Car by searching for a User by ID number.